indicator libraries: init at 12.10

msteen opened the pull request:
The indicator libraries for GTK2 and GTK3.

The bindings for Mono do not work yet because of some issues with Perl in
gtk-sharp-2 and an Unhandled Exception caused by building with multiple cores:
https://aur.archlinux.org/packages/libindicate-sharp/#comment-290385

Some packages have TODOs for the indicator libraries,
since it is an optional dependency for most packages.
These packages have not been updated by this commit.

Besides the Mono bindings, there is still one open issue. For the packages that use autoconf a warning is given that automake-11.1 and aclocal-11.1 are missing. If the package automake-111x is added to the build inputs it will give a warning about a version mismatch between automake 11.1.5 (the package) and 11.1.6 (automake-111x), telling me to regenerate the files. However regenerating the Makefile will result in many warnings about unsupported syntax, which according to the warning were likely to be GNU extensions. Since the initial warnings are about executables that are not actually used, I chose to ignore them for now, instead of potentially breaking stuff by regenerating the Makefiles.

igsha committed:
numlockx: init at 1.2

cstrahan committed:
nixos-generate-config: detect brcmfmac requirement

This makes the firmware available (or would, if someone switched off
enableAllFirmware). Corresponding kernel module should get auto-loaded.

See #9948. Close #9971.

rinaldus committed:
wine-{unstable,staging}: update 1.7.48 -> 1.7.52

Close #10238.

Mathnerd314 committed:
Package all of George Douros's fonts, not just Symbola

puffnfresh committed:
leela: init at 12.fe7a35a

globin committed:
poezio: fix plugin dependencies

abbradar committed:
urxvt_perls: 2015-03-28 -> 2.2

abbradar committed:
yed: 3.14.3 -> 3.14.4

abbradar committed:
rxvt_unicode: add patch to fix Shift+PgUp/PgDown bug

abbradar committed:
pidginlatex: use new texlive

abbradar committed:
pidginlatexSF: move to deprecated aliases

abbradar committed:
libbluray: 0.8.1 -> 0.9.0

abbradar committed:
ipopt: 3.12.3 -> 3.12.4

abbradar committed:
cppzmq: 2015-07-06 -> 20150926

abbradar committed:
folly: 2015-09-17 -> 0.57.0

peti committed:
hackage-packages.nix: update Haskell package set

This update was generated by hackage2nix v20150922-17-gacb24c5 using the following inputs:

  - Nixpkgs: https://github.com/NixOS/nixpkgs/commit/d16b4a3e789f1abb769c6816b9a78466dcc81f8e
  - Hackage: https://github.com/commercialhaskell/all-cabal-hashes/commit/bf464d875465fd575aad1adf660ff83d35ba0a7b
  - LTS Haskell: https://github.com/fpco/lts-haskell/commit/0d493eae3616d7da684f1b6bf35709bc4408a5c0
  - Stackage Nightly: https://github.com/fpco/stackage-nightly/commit/a19cf709b396706a63c627959a6ff52a6c14e8d7

peti committed:
haskell-conduit: fix build with pre-7.10.x versions of GHC

Thanks to @bennofs for finding this issue!

brodul committed:
scrolls: 2014-03-08 -> 2015-10-13

nckx committed:
pythonPackages.buttersink 0.6.6 -> 0.6.7

pSub committed:
perl-Test-Spec: 0.47 -> 0.51

pSub committed:
perl-Test-Shared-Fork: 0.29 -> 0.34

pSub committed:
perl-Test-Requires: 0.06 -> 0.10

pSub committed:
perl-Test-Pod: 1.48 -> 1.51

edolstra committed:
syslinux: Strip static libraries

This prevents a retained dependency on gcc.

dochang committed:
git-crypt: update maintainer

jagajaga committed:
Merge pull request #10387 from dochang/update-maintainer

git-crypt: update maintainer

fps committed:
ardour: fix the git revision and hash to _really_ pull the latest 4.2 version

jagajaga committed:
yandex-disk: update 0.1.5.905 -> 0.1.5.940

rnhmjoj committed:
pyquery: 1.2.4 -> 1.2.9

jagajaga committed:
Merge pull request #10380 from brodul/scrolls

scrolls: update binary source hash

shdpl committed:
dex: init at 0.7

edolstra committed:
Factor out option renaming

Option aliases/deprecations can now be declared in any NixOS module,
not just in nixos/modules/rename.nix. This is more modular (since it
allows for example grub-related aliases to be declared in the grub
module), and allows aliases outside of NixOS (e.g. in NixOps modules).

The syntax is a bit funky. Ideally we'd have something like:

  options = {
    foo.bar.newOption = mkOption { ... };
    foo.bar.oldOption = mkAliasOption [ "foo" "bar" "newOption" ];
  };

but that's not possible because options cannot define values in
*other* options - you need to have a "config" for that. So instead we
have functions that return a *module*: mkRemovedOptionModule,
mkRenamedOptionModule and mkAliasOptionModule. These can be used via
"imports", e.g.

  imports = [
    (mkAliasOptionModule [ "foo" "bar" "oldOption" ] [ "foo" "bar" "newOption" ]);
  ];

As an added bonus, deprecation warnings now show the file name of the
offending module.

Fixes #10385.

jagajaga committed:
djmount: init at 0.71

jagajaga committed:
djmount: add missing file

vbgl committed:
js_of_ocaml: fix for OCaml < 4.0

vbgl committed:
ocaml-ocp-indent: 1.4.2b -> 1.5.2

vbgl committed:
ocaml-cmdliner: 0.9.7 -> 0.9.8

vbgl committed:
ocp-index: requires OCaml >= 4

utdemir committed:
jenkins: 1.631 -> 1.633

expipiplus1 committed:
neovim: 2015-10-08 -> 2015-10-12

expipiplus1 committed:
libtermkey: 0.17 -> 0.18

cillianderoiste committed:
Merge pull request #10388 from fps/ardour-update-to-real-4.2

ardour: 4.1 -> 4.2 (fix the git revision and hash to _really_ pull the latest 4.2)

PJBee committed:
- Added myself as a maintainer
- Boilerplate whiskermenu expression

PJBee committed:
Merge remote-tracking branch 'upstream/master' into xfce4-whiskermenu-plugin

Merge upstream.

jagajaga committed:
Merge pull request #10393 from expipiplus1/libtermkey-version

libtermkey: 0.17 -> 0.18

jagajaga committed:
Merge pull request #10392 from expipiplus1/nvim-version

neovim: 2015-10-08 -> 2015-10-12

jagajaga committed:
Merge pull request #10391 from utdemir/jenkins-633

jenkins: 1.631 -> 1.633

jagajaga committed:
Merge pull request #10386 from Shados/fix-kodi

kodi: Fix SSA/ASS subtitle support

vbgl committed:
eliom: requires OCaml >= 4

Commit:
Merge pull request #10187 from NixOS/switch-to-pd

Switch stdenvs to pure-darwin

peti committed:
configuration-hackage2nix.yaml: disable broken builds

teh committed:
Use versioned tarball URL for chkrootkit.

Pointing at the latest version is fragile: Hash updates don't
follow source updates automatically.

wizeman committed:
flashplayer: 11.2.202.521 -> 11.2.202.535

pSub committed:
Merge pull request #10397 from teh/master

Use versioned tarball URL for chkrootkit.

phile314 committed:
uhc: Update to 1.1.9.1

edolstra committed:
Bump minimum required Nix version

Issue #10337.

abbradar committed:
steam-chrootenv: added corefonts

nckx committed:
rsstail 2015-09-06 -> 2.1

jagajaga committed:
Merge pull request #10389 from shdpl/master

dex: init at 0.7

fps committed:
ardour: 4.2 -> 4.3

abbradar committed:
steam-fonts: init at 1

abbradar committed:
steam-fonts: use for chrootenv

nckx committed:
eid-mw 4.1.6 -> 4.1.7

peti committed:
emacs-markdown-mode: add version 2.0-82-gfe30ef7

fps committed:
ardour: add a clarifying comment to patchPhase

peti committed:
emacs-markdown-mode: fix misspelled license attribute

peti committed:
Merge pull request #10398 from phile314/master

uhc: Update to 1.1.9.1

abbradar committed:
onscripter-en: update homepage

abbradar committed:
crawl: 0.16.1 -> 0.16.2

abbradar committed:
android-udev-rules: 20150821 -> 20150920

abbradar committed:
uwsgi: 2.0.11.1 -> 2.0.11.2

abbradar committed:
sdl-jstest: 20150625 -> 20150806

abbradar committed:
tlp: 0.7 -> 0.8

abbradar committed:
update-resolv-conf: 2014-10-03 -> 20141003

svanderburg committed:
androidenv: fix broken, less frequently used command-line tools

prikhi committed:
pencil: v2.0.13 -> v2.0.14

deepfire committed:
vue:  3.2.2 -> 3.3.0

peti committed:
hackage-packages.nix: update Haskell package set

This update was generated by hackage2nix v20150922-18-g676e5bd using the following inputs:

  - Nixpkgs: https://github.com/NixOS/nixpkgs/commit/0d6c2ccabfffd3942f824ac359c8e36e21d4afb6
  - Hackage: https://github.com/commercialhaskell/all-cabal-hashes/commit/4571e32cd95bf42a53c85e7c8a496d584c7b028f
  - LTS Haskell: https://github.com/fpco/lts-haskell/commit/0d493eae3616d7da684f1b6bf35709bc4408a5c0
  - Stackage Nightly: https://github.com/fpco/stackage-nightly/commit/c10e6d05feb8b1d4a9cb2ef479d2a448934b7984

wkennington committed:
kernel: 3.12.48 -> 3.12.49

wkennington committed:
gnupg: 2.1.8 -> 2.1.9

wkennington committed:
grsecurityUnstable: 4.1.7 -> 4.2.3

cstrahan committed:
powertop: fix --auto-tune

See https://lists.01.org/pipermail/powertop/2014-December/001727.html

7c6f434c committed:
Paratype PT: init

pSub committed:
Merge pull request #10402 from prikhi/update-pencil

pencil: v2.0.13 -> v2.0.14

rehno-lindeque committed:
heroku: 3.32.0 -> 3.42.20

wkennington committed:
chromiumBeta: 46.0.2490.52 -> 46.0.2490.64

wkennington committed:
chromium: 45.0.2454.101 -> 46.0.2490.71

MasseGuillaume committed:
kawkab-mono font type

petarb committed:
install ripmime manual into share/man instead of man

pSub committed:
Merge pull request #10403 from rehno-lindeque/heroku-3.42.20

heroku: 3.32.0 -> 3.42.20

shlevy committed:
virt-manager: Fixup path to libvirt's cpu_map.xml

shlevy committed:
libvirt: Fix typo introduced by 556151911edca15737ca4b7fa6199a2b960f924e

peti committed:
Merge pull request #10405 from MasseGuillaume/feature/kawkab-mono

kawkab-mono font type

Commit:
fix libSystemClosure on non-darwin machines

fps committed:
ardour: 4.3 -> 4.4

anderslundstedt committed:
zerotierone: 1.0.4 -> 1.0.5

fps committed:
ardour: merge upstream master

fps committed:
xonotic: 0.8.0 -> 0.8.1

Commit:
re-fix evaluation of libSystemClosure

taku0 committed:
firefox-bin: 41.0.1 -> 41.0.2

Commit:
inherit system for builtin fetchurl in PD

Commit:
fix filepaths for darwin.security_tool

puffnfresh committed:
nixos-generate-config: look at mmc_host for device drivers

I needed to add sdhci_acpi and mmc_block to my initrd modules in order to boot
my Chromebook. Looking under /sys/class/mmc_host/*/device/driver/module will
give us the sdhci_acpi dependency.

domenkozar committed:
Merge pull request #10406 from anderslundstedt/zerotierone

zerotierone: 1.0.4 -> 1.0.5

domenkozar committed:
Merge pull request #10410 from taku0/firefox-bin-41.0.2

firefox-bin: 41.0.1 -> 41.0.2

cillianderoiste committed:
Merge pull request #10409 from fps/xonotic-0.8.0-to-0.8.1

xonotic: 0.8.0 -> 0.8.1

pSub committed:
calibre: 2.40.0 -> 2.41.0

7c6f434c committed:
firefox: 41.0.1 -> 41.0.2

vbgl committed:
ocaml-ctypes: requires OCaml >= 4

garbas committed:
vimPlugins: some vim addons got removed in e206ea5b probably by accident

pSub committed:
radicale: 0.10 -> 1.0.1

pSub committed:
Merge pull request #10412 from puffnfresh/package/leela

leela: init at 12.fe7a35a

vbgl committed:
ocaml-llvm: init at 3.7.0

peti committed:
haskell: update lists of core and non-core packages for older compilers

peti committed:
hackage-packages.nix: update Haskell package set

This update was generated by hackage2nix v20150922-22-g7ae1e4c using the following inputs:

  - Nixpkgs: https://github.com/NixOS/nixpkgs/commit/ecd4f778f13aaf6c0cf738c523c1e426e72a59f7
  - Hackage: https://github.com/commercialhaskell/all-cabal-hashes/commit/873ad146ce6200f5b90739a46889a0e13d2678f8
  - LTS Haskell: https://github.com/fpco/lts-haskell/commit/0c2d9478c40ebf63cd6ae0f19a84ffaeed4fbe1c
  - Stackage Nightly: https://github.com/fpco/stackage-nightly/commit/3f21ac8e0f24870dfadf10d96e32b2ed943efe2e

cillianderoiste committed:
Merge pull request #10408 from fps/ardour-4.3-to-4.4

ardour: 4.2 -> 4.4

shlevy committed:
haskell-hscurses: fix undeclared dependency on ncurses

waern committed:
ghcjs: make it easier to override shims and ghcjs-boot

rickynils committed:
Merge pull request #10423 from waern/ghcjs-parameterise

ghcjs: make it easier to override shims and ghcjs-boot

wkennington committed:
ceph: 0.94.3 -> 0.94.3.2

wkennington committed:
ceph-dev: 9.0.3 -> 9.1.0

wkennington committed:
ceph-git: 2015-09-22 -> 2015-10-16

jagajaga committed:
Merge pull request #10422 from igsha/numlockx

numlockx: init at 1.2

jagajaga committed:
Merge pull request #10413 from puffnfresh/nixos/mmc_host

nixos-generate-config: look at mmc_host for device drivers

Commit:
nodejs-0.10: apply gyp patches to node-gyp in fixup

vbgl committed:
musescore: 2.0.1 -> 2.0.2

igsha committed:
pymetar: init at 0.20

lsix committed:
pkgs.complexity: 0.4 -> 1.3

See https://lists.gnu.org/archive/html/info-gnu/2015-10/msg00004.html
for announcement

kamilchm committed:
goPackages.glide: 0.5.0 -> 0.6.1

aespinosa committed:
ruby: adjust the libPath

matthiasbeyer committed:
refind: 0.8.4 -> 0.9.2

matthiasbeyer committed:
i3: 4.10.4 -> 4.11

matthiasbeyer committed:
kid3: 3.1.2 -> 3.3.0

cillianderoiste committed:
Merge pull request #10428 from vbgl/musescore-2.0.2

musescore: 2.0.1 -> 2.0.2

wizeman committed:
rustcMaster: 2015-09-23 -> 2015-10-17

The build was changed to compile with the bundled LLVM because compiling
against the system's LLVM now fails with compilation errors.

kragniz committed:
Update email address to currently preferred one

cillianderoiste committed:
Merge pull request #10438 from kragniz/kragniz-maintainer-email-address

Update email address to currently preferred one

wkennington committed:
mbedtls: 1.3.11 -> 1.3.14

peti committed:
haskell-configuration-ghc-7.8.x.nix: remove obsolete overrides

These overrides cause an infinite recursion for reasons I don't quite
understand and that don't seem worthwhile to attempt to understand.

wkennington committed:
grub2: 2015-07-27 -> 2015-10-13

Mathnerd314 committed:
hanazono: init at 20141012

gridaphobe committed:
emacsPackagesNg: add avy, hydra. update async, magit, swiper

vbgl committed:
ocaml-x509: init at 0.4.0

X.509 (RFC5280 and RFC6125) handling in OCaml.

Homepage: https://github.com/mirleft/ocaml-x509

vbgl committed:
Merge pull request #10332 from rnhmjoj/cssselect

cssselect: 0.7.1 -> 0.9.1

Axiomatic- committed:
apulse: 0.1.5 -> 0.1.6

jagajaga committed:
Merge pull request #10446 from Axiomatic-/fix/apulse-name-update

apulse: 0.1.5 -> 0.1.6

wizeman committed:
rustfmt: 2015-09-23 -> 2015-10-17

Also convert from fetchgit to fetchFromGitHub.

wizeman committed:
rustRegistry: 2015-09-23 -> 2015-10-18

anderspapitto committed:
i3: 4.10.4 -> 4.11

Mathnerd314 committed:
unifont_upper: init at 8.0.01

telotortium committed:
xorg.xf86-video-ast: 0.98.0 -> 1.1.5 (close #10451)

Fix failure of 0.98.0 to compile with NixOS 15.09 (due to referencing a
symbol `IOADDRESS` that has been removed from X.org drivers).

taku0 committed:
android-sdk: 24.3.4 -> 24.4, android-ndk: r10c -> r10e

globin committed:
flow: 0.14.0 -> 0.17.0

domenkozar committed:
Merge pull request #10433 from igsha/pymetar

pymetar: init at 0.20

domenkozar committed:
Merge pull request #10447 from anderspapitto/i3

i3: 4.10.4 -> 4.11

domenkozar committed:
Merge pull request #10434 from kamilchm/glide

goPackages.glide: 0.5.0 -> 0.6.1

domenkozar committed:
Merge pull request #10436 from matthiasbeyer/update-some-pkgs

i3, kid3, refind: Package updates

domenkozar committed:
Merge pull request #10454 from mayflower/update-flow

flow: 0.14.0 -> 0.17.0

domenkozar committed:
Merge pull request #10435 from aespinosa/fix-ruby-libpath

ruby: adjust the libPath

domenkozar committed:
Merge pull request #10455 from mayflower/fix-poezio-plugin-deps

poezio: fix plugin dependencies

rycee committed:
svtplay-dl: 0.20.2015.09.13 -> 0.20.2015.10.08

rycee committed:
eclipse-plugin-checkstyle: 6.11.0 -> 6.11.1

svanderburg committed:
Merge pull request #10453 from taku0/android_sdk_24.4

android-sdk: 24.3.4 -> 24.4, android-ndk: r10c -> r10e

igsha committed:
fuse-7z-ng: init at git-2014-06-08

Mathnerd314 committed:
lohit-fonts: Update and split into individual font packages

joachifm committed:
electrum: 2.4.3 -> 2.5.1

rycee committed:
eclipse-plugin-testng: 6.9.7 -> 6.9.8

Mathnerd314 committed:
oldsindhi: init at 0.1

wkennington committed:
btrfsProgs: 4.2.1 -> 4.2.2

globin committed:
apacheHttpd: 2.4.16 -> 2.4.17

Mathnerd314 committed:
sampradaya: init at 2015-05-26

Mathnerd314 committed:
marathi-cursive: init at 1.2

peti committed:
pandoc-citeproc: remove obsolete overrides

The spuriously failing test suite is now disabled by cabal2nix.

Mathnerd314 committed:
tai-ahom: init at 2015-07-06

Mathnerd314 committed:
mro-unicode: init at 2013-05-25

jagajaga committed:
Merge pull request #10462 from igsha/fuse-7z-ng

fuse-7z-ng: init at git-2014-06-08

jagajaga committed:
Merge pull request #10459 from joachifm/electrum-2_5_1

electrum: 2.4.3 -> 2.5.1

jagajaga committed:
Merge pull request #10461 from rycee/bump/testng

eclipse-plugin-testng: 6.9.7 -> 6.9.8

jagajaga committed:
Merge pull request #10457 from rycee/bump/checkstyle

eclipse-plugin-checkstyle: 6.11.0 -> 6.11.1

jagajaga committed:
Merge pull request #10456 from rycee/svtplay-dl

svtplay-dl: 0.20.2015.09.13 -> 0.20.2015.10.08

cpages committed:
yafc: update to 1.3.6

Mathnerd314 committed:
signwriting: init at 1.1.4

fpletz committed:
libressl: 2.2.3 -> 2.2.4

This release fixes a buffer overrun and a memory leak.

See: http://ftp.openbsd.org/pub/OpenBSD/patches/5.8/common/007_obj2txt.patch.sig

vbgl committed:
Merge pull request #10333 from rnhmjoj/pyquery

pyquery: 1.2.4 -> 1.2.9

jagajaga committed:
Merge pull request #10469 from deepfire/vue

vue:  3.2.2 -> 3.3.0

jagajaga committed:
Merge pull request #10467 from mayflower/package/libressl-2.2.4

libressl: 2.2.3 -> 2.2.4

wkennington committed:
git: 2.5.3 -> 2.6.2

wkennington committed:
spice-protocol: 0.12.9 -> 0.12.10

wkennington committed:
spice: 0.12.5 -> 0.12.6

nckx committed:
netsniff-ng 0.5.9-{106-g895377c -> 110-gfc4646b}

wkennington committed:
linuxFirmware: 2015-09-07 -> 2015-10-18

wkennington committed:
mongodb: 3.0.6 -> 3.0.7

Use the vendored wiredtiger as the system one is not compatible

abbradar committed:
grub4dos: init at 0.4.6a

wkennington committed:
picocom: 2.0 -> 2.1

nckx committed:
cassandra 2.1.10 -> 2.1.11

Mathnerd314 committed:
akkadian: fix hash

Mathnerd314 committed:
unidings: fix hash

vcunat committed:
xorg.xf86-input-libinput: update 0.12.0 -> 0.14.0

vcunat committed:
xorg.xorg-server: use the generated expression

... to avoid future surprises. Also add libunwind to deps.
I don't know why libxshmfence is no longer detected.

vcunat committed:
xorg.xf86-video-sis: minor update 0.10.7 -> 0.10.8

vcunat committed:
xorg: add three missing video drivers - chips, s3virge, xgi

Only building was tested.

vcunat committed:
xorg: adjust generator to work with other archive types

In the end I didn't utilize it, but it shouldn't hurt.

abbradar committed:
Merge pull request #10202 from abbradar/llvm-debug

llvm: add debug builds support

abbradar committed:
Merge pull request #10320 from abbradar/fhs-simplify

chroot-env: simplify, clean directories structure

edolstra committed:
Merge pull request #10463 from mayflower/update-apache-httpd

apacheHttpd: 2.4.16 -> 2.4.17

lsix committed:
gnuchess: 6.2.1 -> 6.2.2

See http://lists.gnu.org/archive/html/info-gnu/2015-10/msg00005.html

domenkozar committed:
Merge pull request #10475 from lancelotsix/update_gnuchess

gnuchess: 6.2.1 -> 6.2.2

abbradar committed:
pam_pgsql: init at 0.7.3.2

ttuegel committed:
Merge pull request #10470 from Mathnerd314/unicode-coverage

Improve Unicode coverage

odi committed:
python-packages: pafy 0.3.74 -> 0.4.1

nckx committed:
geolite-legacy 2015-10-13 -> 2015-10-19

dasjoe committed:
service.asterisk: fix dir creation

dasjoe committed:
asterisk: 13.3.2 -> 13.6.0

Commit:
disable gdk-pixbuf test build on darwin

rycee committed:
perl-TextCSVEncoded: init at 0.24

PJBee committed:
Added the xfce4-whiskermenu-plugin

PJBee committed:
Created the xfce4-whiskermenu-plugin expression.

jagajaga committed:
Merge pull request #10480 from dasjoe/asterisk

asterisk: fix service installation and upgrade to 13.6.0

PJBee committed:
Merge branch 'master' of https://github.com/NixOS/nixpkgs into xfce4-whiskermenu-plugin

Pull from nixpkgs/master to resolve Travis CI build conflicts.

willtim committed:
mendeley: 1.13.1 -> 1.15

layus committed:
python-docker-py: fix dependency: requests -> requests2

andrewrk committed:
obs-studio: 0.11.1 -> 0.12.0

Also enable PulseAudio support if unspecified.

Commit:
go-1.5: fix build on darwin, set to default version for all platforms

Commit:
add vim-vanilla changes to vim_configurable

bdimcheff committed:
cups-filters: make shell string longer (close #10493)

The cstring for the shell path is too short for nixos in cups-filters,
causing it to be truncated.  This was previously fixed in #5428, but
regressed.

This is a permanent solution accepted even upstream
https://bugs.linuxfoundation.org/show_bug.cgi?id=1325
http://bzr.linuxfoundation.org/loggerhead/openprinting/cups-filters/revision/7401

Commit:
rethinkdb: build on darwin

dochang committed:
yank: init at 0.4.1

vbgl committed:
swiften: fix build

jagajaga committed:
Merge pull request #10489 from andrewrk/obs-studio

obs-studio: 0.11.1 -> 0.12.0

jagajaga committed:
Merge pull request #10494 from dochang/yank

yank: init at 0.4.1

jagajaga committed:
Merge pull request #10484 from willtim/master

mendeley: 1.13.1 -> 1.15

jagajaga committed:
Merge pull request #10485 from layus/fix1

python-docker-py: fix dependency: requests -> requests2

jagajaga committed:
Merge pull request #10482 from rycee/package/perl-TextCSVEncoded

perl-TextCSVEncoded: init at 0.24

jagajaga committed:
Merge pull request #10479 from odi/master

python-packages: pafy 0.3.74 -> 0.4.1

lsix committed:
pkgs.units: 2.11 -> 2.12

See http://lists.gnu.org/archive/html/info-gnu/2015-10/msg00006.html

vbgl committed:
Merge pull request #10495 from lancelotsix/update_gnu_units

pkgs.units: 2.11 -> 2.12

rickynils committed:
google-cloud-sdk: 0.9.74 -> 0.9.82

Commit:
fix reference to glibtoolize in tmate

Commit:
Merge pull request #9256 from FRidh/networkx

python-packages networkx: 1.9.1 -> 1.10

abbradar committed:
Merge pull request #8299 from abbradar/steam-native

Steam: use native NixOS libraries instead of Steam Runtime

Commit:
darwin: add needed inputs to python-3.4

Commit:
hplip: fix evaluation

Commit:
bluejeans: fix evaluation on non-linux

nckx committed:
gptfdisk 1.0.0 -> 1.0.1

Changes: http://www.rodsbooks.com/gdisk/revisions.html

vbgl committed:
goffice: 0.10.22 -> 0.10.24

wkennington committed:
tmux: 2.0 -> 2.1

domenkozar committed:
Merge pull request #10483 from pjbarnoy/xfce4-whiskermenu-plugin

xfce4-whiskermenu-plugin

domenkozar committed:
Merge pull request #10432 from lancelotsix/update_pkgs.complexity

pkgs.complexity: 0.4 -> 1.3

demin-dmitriy commented on pkgs/development/libraries/libindicator/default.nix:
```diff
@@ -0,0 +1,48 @@
+
+  postPatch = ''
+    substituteInPlace configure \
+      --replace 'LIBINDICATOR_LIBS+="''$LIBM"' 'LIBINDICATOR_LIBS+=" ''$LIBM"'
```
Btw, there is no need in escaping single `$`. For example: `''$abc'' == ''''$abc''` and `''''${abc}'' == "\${abc}"`, but `''${abc''` is syntax error.

msteen replied:
You are right, it is unnecessary here. Thanks for pointing it out to me! I have updated the commit to remove the unnecessary escaping of the dollar signs.

domenkozar commented:
`error: attribute ‘msteen’ missing, at /home/travis/build/NixOS/nixpkgs/pkgs/development/libraries/indicator-application/gtk2.nix:51:21`

domenkozar commented:
You have to add yourself as maintainer to `lib/maintainers.nix`

msteen committed:
Added msteen to the list of maintainers.

msteen commented:
@domenkozar I know, I did this with my first pull requests, which has not been merged yet. I just created a new pull request #10791 that only adds myself to the list of maintainers.

domenkozar commented:
Sorry for confusion, github makes it hard to make PRs that depend on each other.

msteen commented:
Please note that this pull request is still based on 15.09. Let me know if it needs to be rebased on master instead.

domenkozar commented:
Yes, please base any work on top of `master`

msteen committed:
indicator libraries: init at 12.10

The indicator libraries for GTK2 and GTK3.

The bindings for Mono do not work yet because of some issues with Perl in
gtk-sharp-2 and an Unhandled Exception caused by building with multiple cores:
https://aur.archlinux.org/packages/libindicate-sharp/#comment-290385

Some packages have TODOs for the indicator libraries,
since it is an optional dependency for most packages.
These packages have not been updated by this commit.
